take-your-first-steps-with-rust/get-started-with-rust/includes/2-install.md

```diff
@@ -1,3 +1,24 @@
 # Install Rust
 
-< Unit Text >
\ No newline at end of file
+The recommended way to install Rust is to use `rustup`, the Rust Toolchain Installer. Go to
+<https://rustup.rs> to find the appropriate instructions for your operating system.
+
```

Comment:
Could you add a screenshot at this point? I attached one here for reference that you can use.

![rustup](https://user-images.githubusercontent.com/10103121/98585671-7dfdb480-227c-11eb-9078-7a303fe0da07.png)

Note that screenshots typically need to include the full window of the browser or whatever is being captured.

Then you can put the image in the `media/` folder and reference it in this file like:
:::image type="content" source="../media/rustup.png" alt-text="Image of rustup installer website.":::

Reply:
Could you add a bit more direction for the installation?  For example:
```suggestion
Copy the curl command by selecting the clipboard icon and then open up your computer's Terminal or Command Prompt to paste the command.

Wait a few minutes for the installation to complete.
```